### PR TITLE
docs(site): fix typos

### PR DESCRIPTION
**Pull request by huanjani**
Fixed some typos I found while reading through the docs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: site/content/docs/concepts/applications.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-An application is a group of related services, environments, and pipelines. Whether you have one service that does everything or a constelation of micro-services, Copilot organizes them and the environments they're deployed to into an "application".
+An application is a group of related services, environments, and pipelines. Whether you have one service that does everything or a constellation of micro-services, Copilot organizes them and the environments they're deployed to into an "application".
```
Comment (efekarakus) [inline site/content/docs/concepts/applications.md:1]: 🤣 paging @kohidave

File: site/content/docs/concepts/pipelines.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 
 Using an automated release tool like CodePipeline helps make your release manageable. Even if your release isn't particularly complicated, knowing that you can just `git push` to deploy your change always feels a little magical 🌈.
 
 ## Pipeline structure
 
 Copilot can set up a CodePipeline for you with a few commands - but before we jump into that, let's talk a little bit about the structure of the pipeline we'll be generating. Our pipeline will have the following basic structure:
 
-1. __GitHub Source__ - when you push to a configured branch (master by default), a new pipeline execution is triggered.
+1. __GitHub Source__ - when you push to a configured branch (main by default), a new pipeline execution is triggered.
```
Comment (efekarakus) [inline site/content/docs/concepts/pipelines.md:15]: 🙏

**Review by efekarakus — APPROVED** [review]
Omg this is amazing, thank you so much!
Let's run `mkdocs build -d docs` after this gets merged so that these changes can go live asap :D

**Review by iamhopaul123 — APPROVED** [review]
Thank you!

**Pull request merged**